.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       elixir_version:
         description: Elixir version
         type: string
-        default: "1.14.1"
+        default: "1.15.4-otp-25"
```

Comment:
Changing this to `1.15.4` since it looks like it's based on OTP26 already: https://github.com/erlef/docker-elixir/blob/5ac15c1d2bbdbf051f198e471c93f604f71d87eb/1.15/Dockerfile#L1